plugins/pitchfork.py

```diff
@@ -17,7 +18,11 @@ def pitchfork(self, msg, arg):
         """
         match = re.match(r'@?([\w-]+)(?:\s+(?:down\s+)?to\s+(.+))?$',
                          arg)
-        if match:
+        labhub_util = LabHub("bot")
```

Comment:
No need to create a object here, just use dependency like explained [here](http://errbot.io/en/latest/user_guide/plugin_development/dependencies.html?highlight=dependson). Use what variables you want from Labhub